File: app/src/main/java/com/sdwfqin/quickseed/retrofit/ServerException.java
package com.sdwfqin.quickseed.retrofit;

/**
 * 描述：服务器下发的错误
 *
 * @author zhangqin
 * @date 2018/2/27
 */
public class ServerException extends RuntimeException {

    public int code;

    public ServerException(int code, String message) {
        super(message);
        this.code = code;
    }
}

File: app-kt/src/main/java/io/github/sdwfqin/app_kt/retrofit/ServerException.kt
package io.github.sdwfqin.app_kt.retrofit

/**
 * 描述：服务器下发的错误
 *
 * @author zhangqin
 * @date 2018/2/27
 */
class ServerException(var code: Int, message: String) : RuntimeException(message)